libs/block-editor/src/lib/extensions/components/bubble-menu/bubble-menu.component.html

```diff
@@ -1,10 +1,12 @@
 <div class="bubble-menu" (mousedown)="preventDeSelection($event)">
+    <dotcms-bubble-change-dropdown [editor]="editor"></dotcms-bubble-change-dropdown>
```

Comment:
This is just another component, you shouldn't be passing editor, but instead pure input and outputs just like we do with the other components in this HTML.